package.json

```diff
@@ -42,10 +42,8 @@
     "@fortawesome/fontawesome-svg-core": "1.2.18",
     "@fortawesome/free-solid-svg-icons": "5.8.2",
     "@ngx-translate/core": "11.0.1",
-    "@swimlane/ngx-charts": "11.1.0",
-    "angular-archwizard": "4.0.0",
+    "chart.js": "2.9.3",
```

Comment:
The version 3.0 has some breaking changes. Can you check? We can use that from the start to avoid any major change later.

Reply:
Yes there are https://github.com/chartjs/Chart.js/releases, however it's in pre-release state, should I change version?

Reply:
Yes, it will prevent any changes related to migration that we might be forced to make at a later stage.

Reply:
I will update to full release(after a few months) along with testing utilities.